modules/ROOT/pages/log-points.adoc

```diff
@@ -1,17 +1,94 @@
 = Anypoint Monitoring Log Points
 
-The Anypoint Monitoring log points feature enables you to interactively extract data from running managed APIs. You can specify properties (for example, the host and method in the request header) in your managed API for which to generate logs without having to write code or redeploy your application. This helps you troubleshoot without having to restart the application. 
+== Log Points for Apps
+
+The Log Points feature for apps enables you to log information from connectors in any Mule app in real time. If you are using one of the supported connectors, you can select a log level (debug, info, warn, or error) and generate logs. 
+
+=== Prerequisites
+
+* A https://www.mulesoft.com/anypoint-pricing[Titanium subscription] to Anypoint Monitoring and Mule 3.8.5 or later
+* The `???` permission
+
+=== Supported Connectors
+
+If you are using Mule 3.x, the following connectors are supported:
+
+* Amazon S3
+* Amazon SQS
+* Anypoint MQ
+* DB
+* Email
+* File
+* FTP
+* JMS
+* Microsoft Service Bus
+* Mongo
+* Salesforce
+* SFTP
+* Object store
+* TCP
+* UDP
+* VM Transport
+* WMQ
+
+If you are using Mule 4.x, the following connectors are supported:
+
+* Amazon S3
+* Amazon SQS
+* Anypoint MQ
+* DB
+* Email
+* File
+* FTP
+* JMS
+* Microsoft Service Bus
+* Mongo
+* Salesforce
+* SFTP
+* Object store
+* TCP
+* UDP
+* VM Transport
+* WMQ
```

Comment:
In Mule 4.x this connector is called IBM MQ

Reply:
```suggestion
* IBM MQ
```